### ISSUE TITLE
kubelet drain script fails when there is a daemon set deployed on worker

### ISSUE THREAD

**Issue by alex-slynko** [cloudfoundry-incubator/kubo-release#181]
Draing stderr log
```
 node "0c974061-bad5-4810-bcee-52a300aa76e1" cordoned
 Kubelet drain failed
```

Drain stdout log
```
error: unable to drain node "0c974061-bad5-4810-bcee-52a300aa76e1", aborting command...

There are pending nodes to be drained:
  0c974061-bad5-4810-bcee-52a300aa76e1
error: daemonsets.extensions "my-fabulous-node-agent" is forbidden: User "kubelet-drain" cannot get daemonsets.extensions in the namespace "fabulous-infrastructure": fabulous-node-agent-mgg9t, fabulous-node-agent-mgg9t
```

**Comment by alex-slynko** [comment]
From documentation
```
Drain node "foo", even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet on it.
 kubectl drain foo --force
```

**Comment by tvs** [comment]
Should be fixed as of df25069929c082c6b752a42ba79e0f8834748b95
